lib/process-services-cloud/src/lib/process/process-filters/components/process-filters-cloud.component.spec.ts

```diff
@@ -231,14 +231,11 @@ describe('ProcessFiltersCloudComponent', () => {
         expect(filterClickedSpy).toHaveBeenCalledWith(mockProcessFilters[0]);
     });
 
-    it('should reset the filter when the param is undefined', async () => {
+    it('should reset the filter when the param is undefined', () => {
         const change = new SimpleChange(mockProcessFilters[0], undefined, false);
         component.currentFilter = mockProcessFilters[0];
         component.ngOnChanges({ 'filterParam': change });
 
-        fixture.detectChanges();
-        await fixture.whenStable();
```

Comment:
how this is causing regression?

Reply:
It's not this causing the regression but this code that was removed https://github.com/Alfresco/alfresco-ng2-components/pull/7151/files#diff-5f4a96c5280b132011772938a5f847a6e387b383e16bc2720a96db0845e41639R154